README.md

```diff
@@ -1,7 +1,151 @@
 # Intel RealSense Gazebo ROS plugin
 
 This package is a Gazebo ROS plugin for the Intel D435 realsense camera.
- 
+
+This fork adds gazebo plugins for the IMU found in the Intel D435i realsense camera.
+
+## Usage
+
+Download this package and put it into your catkinws src folder and then compile the catkin package.
+This will generate shared libraries called librealsense_gazebo_plugin.so,  for you which is used in the files you gonna download in the next steps.
+
+Download the two xacro files [```_d435.gazebo.xacro```](https://raw.githubusercontent.com/pal-robotics-forks/realsense/upstream/realsense2_description/urdf/_d435.gazebo.xacro) and [```_d435.urdf.xacro```](https://raw.githubusercontent.com/pal-robotics-forks/realsense/upstream/realsense2_description/urdf/_d435.urdf.xacro) (they are from [pal-robotics-forks/realsense](https://github.com/pal-robotics-forks/realsense/tree/upstream/realsense2_description/urdf)).
+Put those two files in the urdf folder of the package where you want your realsense to be simulated.
+
+**In the file ```_d435.urdf.xacro``` change:**
+
+**line 13** from:
+
+```xml
+<xacro:include filename="$(find realsense2_description)/urdf/urdf_d435.gazebo.xacro"/>
+```
+
+to: (note: replace ```packagename``` the name of your package)
+
+```xml
+<xacro:include filename="$(find packagename)/urdf/urdf_d435.gazebo.xacro"/>
+```
+
```

Comment:
This can be removed and instead we should modify the xacro files in the repo, right? ?

Reply:
Yes, it would be great if the modifications described in lines 29 to 138 would be moved to pal-robotics-forks/realsense. 
It would be best to leave lines 9 to 27 for users (like me) that have IntelRealSense/realsense-ros instead of pal-robotics-forks/realsense (the fork is 191 commits behind).